pkg/flow/gateway/emulator.go

```diff
@@ -0,0 +1,89 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gateway
+
+import (
+	"fmt"
+
+	flow2 "github.com/onflow/flow-cli/pkg/flow"
```

Comment:
Haha these name conflicts are awkward. Maybe use `flowcli` instead of `flow2`?

Reply:
yes for sure, I missed it here but what I do is name the sdk `flowsdk`